README.md

```diff
@@ -57,12 +57,10 @@ Community adapters exist for [CouchDB](https://github.com/search?q=sails+couch&n
 [Amazon RDS](https://github.com/TakenPilot/sails-rds), [DynamoDB](https://github.com/TakenPilot/sails-dynamodb), [Azure Tables](https://github.com/azuqua/sails-azuretables), [RethinkDB](https://github.com/gutenye/sails-rethinkdb) and [Solr](https://github.com/sajov/sails-solr); for various 3rd-party REST APIs like Quickbooks, Yelp, and Twitter, including a configurable generic [REST API adapter](https://github.com/zohararad/sails-rest); plus some [eclectic projects](https://www.youtube.com/watch?v=OmcQZD_LIAE).
 
 <!-- Core adapter logos -->
-<a target="_blank" href="http://www.mysql.com">
-  <img width="75" src="http://www.mysql.com/common/logos/powered-by-mysql-125x64.png" alt="Powered by MySQL" title="sails-mysql: MySQL adapter for Sails"/>
-</a>&nbsp; &nbsp; &nbsp; &nbsp;
-<a target="_blank" href="http://www.postgresql.org/"><img width="50" title="PostgreSQL" src="http://i.imgur.com/OSlDDKv.png"/></a>&nbsp; &nbsp; &nbsp; &nbsp;
-<a target="_blank" href="http://www.mongodb.org/"><img width="100" title="MongoDB" src="http://i.imgur.com/bC2j13z.png"/></a>&nbsp; &nbsp; &nbsp; &nbsp;
-<a target="_blank" href="http://redis.io/"><img width="75" title="Redis" src="http://i.imgur.com/dozv0ub.jpg"/></a>&nbsp; &nbsp; &nbsp; &nbsp;
+<a target="_blank" href="http://www.mysql.com/"><img width="55" title="MySQL" src="https://cdn.jsdelivr.net/gh/devicons/devicon@latest/icons/mysql/mysql-original-wordmark.svg" alt="MySQL"/></a>&nbsp; &nbsp; &nbsp; &nbsp;
```

Comment:
Hello @dushmanta05 
Good PR 👍 
The changes are misleading as the PR title meant to update only logo URLs. But, the content is changed.
I noticed the MySQL is changed from 
`alt="Powered by MySQL" title="sails-mysql: MySQL adapter for Sails"`
to 
`alt="MySQL" title="MySQL"`

Either change PR title to match the changes or Refactor the changes to match PR title.

Reply:
Thank you. I changed the title to make it uniform with other logos and titles; I'll update the PR title. Thanks again.